components/create/basic.tsx

```diff
@@ -47,7 +51,44 @@ function Basic() {
           />
         </div>
         <ErrorMessage name="walletName" render={renderError} />
-        <div className="mt-8 flex space-x-6">
+        <div className="mt-6">
+          <label className="font-medium text-white">Select Version</label>
+          <div className="mt-2">
+            <label className="flex items-center text-white">
+              <Field
+                type="radio"
+                name="version"
+                value="0.3.3"
+                className="mr-2 "
+                checked
+              />
+              <div>v0.3.3</div>
+            </label>
+            <div className="ml-2 text-sm text-gray-500">
+              {
+                "This version doesn't support for TZIP27 in processing proof-of-event challenges for message signing. If such functionality is unnecessary, this version is recommended. The creation fee is roughly 1.8 tez, depending on the count of owners."
```

Comment:
Not sure how I would say it, but this is unclear imo, we should find a way to say it that is closer to what the user can understand